### DISCUSSION TITLE
运行了2天，worker一直处于等待任务状态。开始惩罚扣币了。任务卡在 PreCommitWait ，请问有遇到过类似的吗？

### DISCUSSION THREAD

**Discussion by k178613133** [filecoin-project/lotus#4671] [Storage Provider] [answered]
1. worker一直处于等待任务状态，没有接收到miner分发的任务
2020-10-31T03:47:06.935+0800    INFO    main    lotus-seal-worker/main.go:322   Opening local storage; connecting to master
2020-10-31T03:47:07.376+0800    INFO    main    lotus-seal-worker/main.go:372   Setting up control endpoint at 127.0.0.1:3456
2020-10-31T03:47:07.409+0800    INFO    main    lotus-seal-worker/main.go:436   Waiting for tasks

2. 运行 lotus-miner sectors list。4个任务卡在了 PreCommitWait ，这种任务状态是否不正常？

ID  State          OnChain  Active  Expiration  Deals
0   PreCommitWait  NO       NO      n/a         CC
0   PreCommitWait  NO       NO      n/a         CC
1   PreCommitWait  NO       NO      n/a         CC
6   PreCommitWait  NO       NO      n/a         CC
7   PreCommit1     NO       NO      n/a         CC
8   PreCommit1     NO       NO      n/a         CC
9   PreCommit1     NO       NO      n/a         CC

3. lotus-miner sealing jobs PC1状态请问一般需要多少时间？目前快7个小时了，还在进行中
ID  Sector  Worker  Hostname  Task  State    Time
3   9       0       smart681  PC1   running  6h45m13.7s
4   8       0       smart681  PC1   running  6h45m13.7s
5   7       0       smart681  PC1   running  6h45m13.7s

**Comment by k178613133** [comment, marked as answer]
1. 删除worker的存储目录。
2. lotus-miner sectors remove 删除对应的任务
3. 关闭 worker
4. 关闭 miner
5. 配置miner只执行 AddPiece任务和unseal任务
4. 重启miner，重新获取新扇区任务
5. 重启worer，添加运行参数，屏蔽 AddPiece和unseal任务
6. 拉取任务

**Comment by jennijuju** [comment]
Converting to discussion as this is a question instead of a bug report.
